lib/android/app/src/main/java/com/reactnativenavigation/parse/TopTabOptions.java

```diff
@@ -1,15 +1,23 @@
 package com.reactnativenavigation.parse;
 
+import android.graphics.Typeface;
+import android.support.annotation.Nullable;
+
+import com.reactnativenavigation.utils.TypefaceLoader;
+
 import org.json.JSONObject;
 
 public class TopTabOptions implements DEFAULT_VALUES {
     public String title = NO_VALUE;
+    @Nullable public Typeface fontFamily;
+    public int tabIndex;
 
-    public static TopTabOptions parse(JSONObject json) {
+    public static TopTabOptions parse(TypefaceLoader typefaceManager, JSONObject json) {
         TopTabOptions result = new TopTabOptions();
         if (json == null) return result;
 
         result.title = json.optString("title", NO_VALUE);
+        result.fontFamily = typefaceManager.getTypeFace(json.optString("titleFontFamily"));
```

Comment:
Why do you need to load typeface now. I mean it's better to have it consistent. Because topBar options load typeface in presenter, and topTabs in parser.